src/components/account/AccountLogin.tsx

```diff
@@ -54,24 +60,44 @@ const useStyles = makeStyles(theme => ({
   },
 }))
 
-const AccountLogin: FunctionComponent<AccountLoginProps> = ({isArcSignIn}) => {
+const AccountLogin: FunctionComponent<AccountLoginProps> = ({
+  isArcSignIn,
+  submitUsernameAndPassword,
+  isLoadingLoginWithUsernameAndPassword,
+  errorMessageLoginWithUsernameAndPassword,
+}) => {
   //const [alertMsg, setAlertMsg] = useState<{msg: string; type: AlertProps['severity']} | undefined>()
-
+  const sourceApp = useSourceApp()
```

Comment:
New hook to get the info about this source app, e.g. long name, short name, etc... We can use this hook when we tackle [DHP-1011](https://sagebionetworks.jira.com/browse/DHP-1011).

Reply:
Can you and a TODO comment (signed and dated) with the JIRA ticket? Thanks!

Reply:
Added TODO in `types/sourceAppConfig.ts`